src/server/modelManager.js

```diff
@@ -1,5 +1,48 @@
+const Grid = require("./models/grid");
+const Frame = require('./frame');
+
+
+var grid;
+
 function createGrid(json)
 {
+    let nbFrame=json.nbFrame;
+    let nbKeel=json.nbKeel;
+    let players=json.players;
+    grid=new Grid(nbKeel,nbFrame);
+    players.forEach(element => {
+      grid.addPlayer(element);
+    });
+    grid.players[players[0]].isPlayin=true;
+}
 
+// A modifier pour faire évoluer le joueur en cours et la manche en cours
```

Comment:
Prefer adding a TODO at the start of comment like that in order to be sure to fix it (it will create a warning if a comment start by` // TODO : `

Reply:
Done